docs/api-reference/core/resources/texture.md

```diff
@@ -54,38 +54,38 @@ Usage expresses two things: The type of texture and what operations can be perfo
 
 Note that the allowed combinations are very limited, especially in WebGPU.
 
-| Usage Flag             | Value  | Description                                              |
-| ---------------------- | ------ | -------------------------------------------------------- |
-|  `Texture.COPY_SRC` | 0x01 | |
-|  `Texture.COPY_DST` | 0x02 | |
-|  `Texture.TEXTURE_BINDING` | 0x04 | |
-|  `Texture.STORAGE_BINDING` | 0x08 | |
-|  `Texture.RENDER_ATTACHMENT` | 0x10 | |
+| Usage Flag                  | Value | Description |
+| --------------------------- | ----- | ----------- |
+| `Texture.COPY_SRC`          | 0x01  |             |
+| `Texture.COPY_DST`          | 0x02  |             |
+| `Texture.TEXTURE_BINDING`   | 0x04  |             |
+| `Texture.STORAGE_BINDING`   | 0x08  |             |
+| `Texture.RENDER_ATTACHMENT` | 0x10  |             |
 
 ## TextureDimension
 
 | Dimension    | WebGPU | WebGL2 | WebGL1 | Description                                                          |
```

Comment:
Shouldn't the WebGL1 column be removed?

Reply:
Good catch, will address in follow-up PR